### DISCUSSION TITLE
How to activate evil_lualine

### DISCUSSION THREAD

**Discussion by Trid-collab** [nvim-lualine/lualine.nvim#546] [Q&A]
I am new to lualine. Wanted some help on how to activate 'evil_lualine' shown at site. While I try  doing `require("lualine").setup({ options = { theme = "tokyonight" } })
` it gives me an error mentioning that the theme does'nt exist

**Comment by shadmansaleh** [comment]
> Wanted some help on how to activate 'evil_lualine' shown at site

evil_line config is provided as [example](https://github.com/nvim-lualine/lualine.nvim/blob/master/examples/evil_lualine.lua) in examples folder .

> require("lualine").setup({ options = { theme = "tokyonight" } })

You need to have tokyonight colorscheme installed and need to load it before lualine .

**Comment by taizilongxu** [comment]
Is there anyway to set the example in another file and import in config.lua?
Cannot find how to

**Comment by ddogfoodd** [comment]
As a new nvim and lualine user, I as well wonder how to activate the examples (e.g. evil_lualine).

I see, those are files in the [example dir ](https://github.com/nvim-lualine/lualine.nvim/blob/master/examples/), but where do I put them?

**Reply by ddogfoodd** [reply]
I finally found out how to use an example file given in the repo.

1. Download or copy the example file that you want to use (and modify it if wanted).
2. Put the file in `~/.config/nvim/lua/plugins/lualine/<example_file>.lua`, create those directories if you don't have them yet.
3. To your nvim config (e.g. `~/.config/nvim/init.lua`) add the line `require 'plugins.lualine.<example_file>'` somewhere to the end or after the other lualine stuff you probably have in there.
